awx/ui/client/src/instance-groups/main.js

```diff
@@ -51,7 +51,7 @@ angular.module('instanceGroups', [CapacityBar.name])
             }
 
             $stateProvider.state({
-                name: 'instanceGroups',
+                name: 'instanceGroups.**',
```

Comment:
Is the `name.**` part documented somewhere in the ui-router docs? I can't seem to find information on that syntax (and others I know we've used).

Reply:
@gconsidine https://ui-router.github.io/ng1/docs/latest/interfaces/ng1.ng1statedeclaration.html#lazyload 

```
A future state can act as a placeholder for a single state, or for an entire module of states and
substates. A future state should have:

A name which ends in .**. A future state's name property acts as a wildcard Glob. It matches
any state name that starts with the name (including child states that are not yet loaded).
```